src/20241104_AaveV3Ethereum_OnboardRsETHToAaveV3Ethereum/AaveV3Ethereum_OnboardRsETHToAaveV3Ethereum_20241104.sol

```diff
@@ -0,0 +1,131 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {AaveV3Ethereum, AaveV3EthereumAssets} from 'aave-address-book/AaveV3Ethereum.sol';
+import {AaveV3PayloadEthereum} from 'aave-helpers/src/v3-config-engine/AaveV3PayloadEthereum.sol';
+import {EngineFlags} from 'aave-v3-origin/contracts/extensions/v3-config-engine/EngineFlags.sol';
+import {IAaveV3ConfigEngine} from 'aave-v3-origin/contracts/extensions/v3-config-engine/IAaveV3ConfigEngine.sol';
+import {IERC20} from 'solidity-utils/contracts/oz-common/interfaces/IERC20.sol';
+import {SafeERC20} from 'solidity-utils/contracts/oz-common/SafeERC20.sol';
+import {IEmissionManager} from 'aave-v3-origin/contracts/rewards/interfaces/IEmissionManager.sol';
+/**
+ * @title Onboard rsETH to Aave V3 Ethereum
+ * @author ACI
+ * - Snapshot: https://snapshot.org/#/aave.eth/proposal/0xe83b67dfdddd469c298ce6133f4fdb84c9796c671c023b88617d5a25b5933c7f
+ * - Discussion: https://governance.aave.com/t/arfc-add-rseth-to-aave-v3-ethereum/17696
+ */
+contract AaveV3Ethereum_OnboardRsETHToAaveV3Ethereum_20241104 is AaveV3PayloadEthereum {
+  using SafeERC20 for IERC20;
+
+  address public constant rsETH = 0xA1290d69c65A6Fe4DF752f95823fae25cB99e5A7;
+  uint256 public constant rsETH_SEED_AMOUNT = 4e16;
+  address public constant rsETH_ADMIN = 0xac140648435d03f784879cd789130F22Ef588Fcd;
+
+  function _postExecute() internal override {
+    IERC20(rsETH).forceApprove(address(AaveV3Ethereum.POOL), rsETH_SEED_AMOUNT);
+    AaveV3Ethereum.POOL.supply(rsETH, rsETH_SEED_AMOUNT, address(AaveV3Ethereum.COLLECTOR), 0);
+
+    (address arsETH, , ) = AaveV3Ethereum.AAVE_PROTOCOL_DATA_PROVIDER.getReserveTokensAddresses(
+      rsETH
+    );
+    IEmissionManager(AaveV3Ethereum.EMISSION_MANAGER).setEmissionAdmin(rsETH, rsETH_ADMIN);
+    IEmissionManager(AaveV3Ethereum.EMISSION_MANAGER).setEmissionAdmin(arsETH, rsETH_ADMIN);
+  }
+
+  function capsUpdates() public pure override returns (IAaveV3ConfigEngine.CapsUpdate[] memory) {
+    IAaveV3ConfigEngine.CapsUpdate[] memory capsUpdate = new IAaveV3ConfigEngine.CapsUpdate[](2);
+
+    capsUpdate[0] = IAaveV3ConfigEngine.CapsUpdate({
+      asset: AaveV3EthereumAssets.wstETH_UNDERLYING,
+      supplyCap: EngineFlags.KEEP_CURRENT,
+      borrowCap: 60_000
+    });
+    capsUpdate[1] = IAaveV3ConfigEngine.CapsUpdate({
+      asset: AaveV3EthereumAssets.ETHx_UNDERLYING,
+      supplyCap: EngineFlags.KEEP_CURRENT,
+      borrowCap: 5_000
+    });
+
+    return capsUpdate;
+  }
+  function eModeCategoriesUpdates()
+    public
+    pure
+    override
+    returns (IAaveV3ConfigEngine.EModeCategoryUpdate[] memory)
+  {
+    IAaveV3ConfigEngine.EModeCategoryUpdate[]
+      memory eModeUpdates = new IAaveV3ConfigEngine.EModeCategoryUpdate[](1);
+
+    eModeUpdates[0] = IAaveV3ConfigEngine.EModeCategoryUpdate({
+      eModeCategory: 2,
+      ltv: 92_50,
+      liqThreshold: 94_50,
+      liqBonus: 1_00,
+      label: 'rsETH'
```

Comment:
```suggestion
      label: 'rsETH LST main'
```
To be consistent, I would add the label as `rsETH LST main`

Reply:
I preferred to rename it has "rsETH-Correlated", it makes more sens here (+ it has been renamed also liked this in the forum post)

Reply:
Indeed, there's only 1 "ETH Correlated" e-mode, the other ones use the format you proposed. So yes it makes more sens to follow this format okay 👍